src/status_im/contexts/wallet/create_account/view.cljs

```diff
@@ -63,76 +59,83 @@
         derivation-path       (reagent/atom (utils/get-derivation-path number-of-accounts))
         {:keys [public-key]}  (rf/sub [:profile/profile])
         on-change-text        #(reset! account-name %)
-        [primary-name _]      (first (rf/sub [:contacts/contact-two-names-by-identity public-key]))
+        primary-name          (first (rf/sub [:contacts/contact-two-names-by-identity public-key]))
```

Comment:
should this not be another sub that does this `first` instead? I think we generally want to do all the data accessing within the subs

Reply:
@J-Son89 is it worth it being a sub. This sub already gets minimalistic data (full name), but here we need the first name to display it as "User's keypair"

Reply:
That I'm not sure of :) - I guess with the subs being tested we have the advantage of having some security what is rendered (provided we don't modify data outside of a sub). For me it's fine either way 👍